chapters/CRUD-Operations.md

```diff
@@ -163,7 +163,7 @@ The steps are as follows:
 1. Else If `identifierComponents.idType` value is "external", then set the
    `initialDocument` to the result of running [External Resolution] passing in
    the `identifier`, `identifierComponents` and `resolutionOptions` values.
-1. Else MUST raise `invalidHRPValue` error.
+1. Else MUST raise ::https://www.w3.org/ns/did#INVALID_DID:: error.
```

Comment:
```suggestion
1. Else MUST raise ::https://www.w3.org/ns/did#INVALID_DID:: error.
```
Again, I am afraid it needs to just be a normal markdown link syntax

Do you know how to render the spec locally?

Reply:
"scripts": {
    "pandoc-spec": "pandoc-spec",
    "pandoc-spec-watch": "pandoc-spec --watch"
  },